<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>analyzeClass</class>
 <widget class="QMainWindow" name="analyzeClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1219</width>
    <height>816</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>analyze</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="cameraListTab">
           <attribute name="title">
            <string>Tab 1</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QToolButton" name="toolButton">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="GraphListWidget" name="graphListlWidget" native="true"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="loginButton"/>
   <addaction name="processButton"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="loginButton">
   <property name="text">
    <string>登陆</string>
   </property>
  </action>
  <action name="processButton">
   <property name="text">
    <string>任务</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GraphListWidget</class>
   <extends>QWidget</extends>
   <header location="global">graphlistwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="analyze.qrc"/>
 </resources>
 <connections/>
</ui>
